cmd/template/cluster/provider/capo.go

```diff
@@ -65,3 +71,97 @@ func WriteOpenStackTemplate(ctx context.Context, client k8sclient.Interface, out
 
 	return nil
 }
+
+func WriteOpenStackTemplateAppCR(ctx context.Context, config ClusterCRsConfig) error {
+	var userConfigConfigMapYaml []byte
+	var err error
+
+	clusterAppOutput := templateapp.AppCROutput{}
+	defaultAppsAppOutput := templateapp.AppCROutput{}
+
+	clusterAppConfig := templateapp.Config{
+		AppName:   fmt.Sprintf("%s-cluster", config.Name),
```

Comment:
@ljakimczuk @tfussell I found a problem. We use `.Release.Name` in the cluster helm chart which turns out badly :(

```
ross3-cluster   Provisioned   4m37s   v1.20.9
```

Should we remove the `-cluster` suffix here or in the helm chart?

Reply:
I guess we could remove it from here, though App name may then be vague to the user, cause in some cases it would be auto-generated